vms/administrator/templates/administrator/report.html

```diff
@@ -147,31 +147,31 @@
             </fieldset>
             <div class="form-group">
                 <div class="col-md-12 col-md-offset-3">
-                    <button class="btn btn-primary" type="submit">Generate Report</button>
+                    <button class="btn btn-primary" type="submit">{% trans "Generate Report" %}</button>
                 </div>
             </div>
         </form>
     </div>
     {% if report_list %}
         <div class="well">
-            <legend>Report Statistics</legend>
-            <label><h4><span class="label label-default">Total Number of Shifts</span></h4></label>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp{{ report_list|length }}
+            <legend>{% trans "Report Statistics" %}</legend>
+            <label><h4><span class="label label-default">{% trans "Total Number of Shifts" %}</span></h4></label>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp{{ report_list|length }}
             <br>
-            <label><h4><span class="label label-default">Total Hours</span></h4></label>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp{{ total_hours }}
+            <label><h4><span class="label label-default">{% trans "Total Hours" %}</span></h4></label>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp{{ total_hours }}
         </div>
         <div class="well">

```

Comment:
@willingc @vubo Doesn't the use of &nbsp makes the page more machine size dependent. or are these fine? I'm not sure. Can we use something else, if not?